projects/plugins/jetpack/extensions/blocks/premium-content/edit.js

```diff
@@ -282,9 +287,19 @@ function Edit( props ) {
 		// Execution delayed with setTimeout to ensure it runs after any block auto-selection performed by inner blocks
 		// (such as the Recurring Payments block)
 		setTimeout( () => props.selectBlock(), 1000 );
-	// eslint-disable-next-line react-hooks/exhaustive-deps
+		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [] );
 
+	const shouldShowConnectButton = () => {
+		if ( ! shouldUpgrade && apiState !== API_STATE_CONNECTED && connectURL ) {
+			return true;
+		}
+
+		return false;
+	};
+
+	const [ , autosaveAndRedirect ] = useAutosaveAndRedirect( connectURL );
```

Comment:
Is something missing here? One of `autosave | isRedirecting` or is the comma not required?

Reply:
I think it's for the array destructuring syntax since the hook returns an array instead of an object. Would it be worth updating the hook to return an object instead, since we aren't using array destructuring to rename the methods?

Reply:
Oh, the opening comma skips the first item. 👍 Thanks!
Carry on 👍

Reply:
I'm skipping the first item (`autosave`) in the array destructuring with the placeholder comma. I'm actually unsure what the convention here should be -- `_autosave` instead to indicate it's unused for the linter?

Reply:
+1 for the object return value!